packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,275 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import Anatomy from './ActionBarAnatomy.svg';
+import docs from 'docs:@react-spectrum/actionbar';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/actionbar/package.json';
+
+```jsx import
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+```
+
+---
+category: Toolbars
+keywords: [actionbar, selection, actions]
+---
+
+# ActionBar
+
+<PageDescription>{docs.exports.ActionBar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionBar, ActionBarContainer, Item']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/action-bar/'}
+  ]} />
+
+## Example
+```tsx example
+function StaticActionBar() {
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        isEmphasized
+        selectedItemCount={3}>
+        <Item key="edit">
+          Edit
+        </Item>
+        <Item key="copy">
+          Copy
+        </Item>
+        <Item key="delete">
+          Delete
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+<StaticActionBar />
+```
+
+## Content
+<Anatomy />
+
+An ActionBar is a companion component intended to facilitate bulk actions on selected items within a collection component. It consists of an ActionGroup, a clear button, and a count of selected items. It accepts a list of <TypeLink links={docs.links} type={docs.exports.Item} />
```

Comment:
`... of an ActionGroup of actions, a clear...`?
Should order of ActionGroup, button, and count mirror the anatomy diagram order?

Reply:
I think the current order is fine since the ActionGroup is the most important part of the component.